cetmix_tower_server/models/cx_tower_command_log.py

```diff
@@ -52,6 +57,11 @@ class CxTowerCommandLog(models.Model):
         store=True,
         index=True,
     )
+    job_pid = fields.Char(
```

Comment:
Is this a queue_job id or ssh_id? If ssh it's better to rename the field into `ssh_jop_pid`.